requirements.txt

```diff
@@ -4,7 +4,7 @@ cachetools~=4.1
 click<9.0
 cookiecutter>=2.1.1, <3.0
 dynaconf>=3.1.2, <4.0
-fsspec>=2021.4, <=2022.1
+fsspec>=2021.4, <=2022.7.1
```

Comment:
Any reason why you pinned it to `2022.7.1` and not just `2022.7`?

Reply:
`2022.7.1` is the latest release that fixes bugs in `2022.7`. Pinning to `<2022.8` wouldn't make sense here since they're following calver.